wallet/src/de/schildbach/wallet/ui/dashpay/NotificationsActivity.kt

```diff
@@ -145,7 +145,7 @@ class NotificationsActivity : InteractionAwareActivity(), TextWatcher,
                             (notificationsAdapter.getItem(currentPosition).notificationItem as NotificationItemContact).usernameSearchResult.toContactRequest = it.data!!
                             notificationsAdapter.notifyItemChanged(currentPosition)
                             currentPosition = -1
-                            lastSeenNotificationTime = it.data.timestamp.toLong()
+                            lastSeenNotificationTime = it.data.timestamp / 1000
```

Comment:
we could store lastSeenNotificationTime in milleseconds to stay consistent with all other Platform timestamps.

Reply:
https://github.com/dashevo/dash-wallet/blob/evonet-develop/wallet/src/de/schildbach/wallet/data/NotificationItemContact.kt#L15

This line which takes a millisecond time and multiplies by 1000 is the problem, probably.  If this is not done, then we don't need to divide by 1000.